test/params_test.c

```diff
@@ -401,7 +401,7 @@ static OSSL_PARAM *construct_api_params(void)
     params[n++] = OSSL_PARAM_construct_utf8_string("p5", app_p5,
                                                    sizeof(app_p5), &app_p5_l);
     params[n++] = OSSL_PARAM_construct_utf8_ptr("p6", (char **)&app_p6,
-                                                &app_p6_l);
+                                                sizeof(app_p6), &app_p6_l);
```

Comment:
The three places in params_test the app_p6 is defined as a param (`static_raw_params`, `static_api_params` and here), the size is set to sizeof(pointer) which is incorrect if it is meant to contain the length of what app_p6 points at.

Reply:
You're quite right...  and also, there setting test ignores p6 completely...  thanks for pointing it out.